README.md

```diff
@@ -5,14 +5,8 @@
 
 Integrate chat completion into your business applications with SAP Cloud SDK for AI. Leverage the generative AI hub of SAP AI Core to make use of templating, grounding, data masking, content filtering and more. Setup your SAP AI Core instance with SAP Cloud SDK for AI Core.
 
-## Disclaimer ⚠️
-
-This project is currently in an experimental state. All functionality and naming is subject to change. Use at your own discretion.
-
 ## Table of Contents
```

Comment:
```suggestion
### Table of Contents
```
At least for this famous VSCode extension `Markdown All in One`, if mark this section with three `#`, it won't auto generate a new entry for `Table of Contents` anymore.